libs/category/src/effects/category.effects.ts

```diff
@@ -142,19 +162,21 @@ export class DaffCategoryEffects {
 		}))
 	)
 	
-	private toggleCategoryFilter(
-		toggledFilter: DaffCategoryFilterAction, 
-		appliedFilters: DaffCategoryFilterAction[]
-	): DaffCategoryFilterAction[] {
-		return appliedFilters.find(filter => this.areFiltersEqual(toggledFilter, filter))
-		// filter is applied, remove it
-		? appliedFilters.filter(filter => !this.areFiltersEqual(filter, toggledFilter))
-		// filter is not applied, add it
-		: appliedFilters.concat(toggledFilter);
-	}
+	validateFilters(filters: DaffCategoryFilterRequest[]): void {
+		if(!filters) return;
+		filters.forEach((filter, i) => {
+			if(filter.type === DaffCategoryFilterType.Range &&
+				filter.value[0].split(DaffCategoryFromToFilterSeparator).length !== 2) {
+				throw new Error('Category range filter is in an invalid format. Should be **-**');
+			}
 
-	private areFiltersEqual(filter1: DaffCategoryFilterAction, filter2: DaffCategoryFilterAction): boolean {
-		return filter1.name === filter2.name && filter1.value === filter2.value;
+			for(let j=i+1; j < filters.length; j++) {
+				if(filters[i].name === filters[j].name) {
+					throw new Error('More than one category filter of the same name exists. These should' +
```

Comment:
This validation isn't necessary and work should be done in the reducer to accommodate this.

Reply:
I forgot about this. Still need to do